edb/ir/ast.py

```diff
@@ -952,11 +953,7 @@ class Call(ImmutableExpr):
     # Bound arguments.
     # Named arguments will come first, sorted by name,
     # followed by positional arguments, in order of declaration.
```

Comment:
This comment is not true anymore, right?

Named arguments don't come before positional ones.

Reply:
Good catch. Did another sweep of the code to check if i missed any other comments.